test/prism/errors_test.rb

```diff
@@ -781,8 +781,8 @@ def test_method_parameters_after_arguments_forwarding
           nil
         ),
         nil,
-        [:"...", :a],
```

Comment:
If we remove these variables, then it might be good to have the new forwarding flags on the various "scope" nodes (DefNode, etc).
Although I'm unsure if needed or not, @andrykonchin probably you know better.
I guess it's not needed because we'd still see these special names e.g. under ParametersNode.
And it probably makes sense to not have them in the local tables since Prism local tables seems to be basically "user-visible variables" (e.g. through Binding#local_variables).

(in any case, not something that needs to be solved in this PR)